index.js

```diff
@@ -0,0 +1,80 @@
+// 1
+
+const movies = ["The Shawshank Redemption",
+    "The Godfather",
+    "The Dark Knight",
+    "12 Angry Men",
+    "The Lord Of The Rings",
+    "Pulp Fiction",
+    "The Good, the Bad and the Ugly",
+    "Forrest Gump"];
+
+    for (let index = 0; index < movies.length; index++) {
+        const element = movies[index];
+        console.log(element);
+       console.log(getWordCount(element))
+    }
+
+function getWordCount(str) {
+        
+     return str.trim().split(/\s+/).length;
+}
+
+// 2
+
+const body = document.querySelector('body');
+const btn = document.querySelector('.js-btn');
+
+body.insertAdjacentHTML('beforeend', `<button class="js-btn">Click</button>`);
+
+
+window.addEventListener('click', onClick);
+
+function onClick(event) {
+    body.insertAdjacentHTML('beforeend', `<h1>Javascript test</h1>`);
+    body.style.backgroundColor = "red";
+    event.target.disabled = true;
+    
+}
+
+// 3
+
+const API = 'https://reqres.in/api/users';
+
+async function fetchApi() {
+    const response = await fetch(API);
+    const data = await response.json();
+    console.log(data)
+    return data;
+
+}
+fetchApi().then(data => renderMarkup(data.data)).catch(error=> console.log(error))
+
+function renderMarkup(arr) {
+    for (let index = 0; index <= 2; index++) {
+          const element = arr[index];
+        const { avatar, first_name } = element;
+        const markup = `
+        <div>
+            <p>${first_name}</p>
+            <img src="${avatar}" alt="">
+        </div>`
```

Comment:
This is definitely a correct way of doing things, especially nice to see you using template literals! (strings with the ' ` ' character)

However, you could have used `document.createElement("p")` and ``document.createElement("img")`` too. However, you solution is not wrong, only more error prone, since one typo can mess up your HTML code.